docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise Cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience.
+
+## Key Benefits
+
+- **Enhanced Audio Quality**: Eliminates unwanted noise, including background chatter, clicks, claps, barking, and other sudden audio disturbances, resulting in a more pleasant listening experience for your audience.
+
+- **Improved Clarity**: Ensures that the primary audio content remains prominent and intelligible by reducing distractions caused by ambient noise.
+
+- **Optimized Communication**: Facilitates seamless communication in conferences and live streams by minimizing disruptions caused by environmental factors, thereby enhancing the overall professionalism of the presentation.
+
+This is a guide to enabling and using the Noise Cancellation on 100ms.
+
+<StepsToc
+    parentId="noise-cancellation"
+    descriptions={[
+        "General information on 100ms Noise Cancellation.",
+        "Enabling the Noise Cancellation for your room(s).",
+        "Exploring platforms and methods of integration.",
+        "Activating and Managing Noise Cancellation."
+    ]}
+/>
+
+<StepsContainer id="noise-cancellation">
+
+### Getting Started
+100ms Noise Cancellation is powered by [krisp.ai](https://krisp.ai/),ensuring clear communication by filtering out background noise.
+
+<Note type='warning'>
+**IMPORTANT** <br/>
+This is an add-on paid feature, for details check [100ms pricing page](https://www.100ms.live/pricing/) 
+</Note>
+
+### Enabling the Noise Cancellation
```

Comment:
We can just say 'Enabling Noise Cancellation'. 
Avoid 'the' unless you follow 'Noise Cancellation' with 'feature'.

So it'll be either :
1. Enabling Noise Cancellation
2. Enabling the Noise Cancellation feature

This way, it would establish the reason for 'the' in the sentence by calling out NC as a specific feature.

Reply:
Also, this is primarily because it isn't incorrect. It just reads rough. Enabling the Noise Cancellation is also technically correct because we're referring to it as a specific feature.